bundles/org.openhab.binding.miio/src/main/resources/database/yunmi.waterpurifier.lx8.json

```diff
@@ -0,0 +1,224 @@
+{
+	"deviceMapping": {
+		"id": [
+			"yunmi.waterpuri.lx8"
+		],
+		"channels": [
+			{
+				"property": "run_status",
+				"friendlyName": "Run Status",
+				"channel": "run_status",
+				"type": "Number",
+				"refresh": true,
+				"actions": []
+			},
+			{
+				"property": "rinse",
+				"friendlyName": "Rinse",
+				"channel": "rinse",
+				"type": "Number",
+				"refresh": true,
+				"actions": []
+			},
+			{
+				"property": "tds_in",
+				"friendlyName": "TDS in",
+				"channel": "tds_in",
+				"type": "Number",
+				"refresh": true,
+				"actions": []
+			},
+			{
+				"property": "tds_out",
+				"friendlyName": "TDS out",
+				"channel": "tds_out",
+				"type": "Number",
+				"refresh": true,
+				"actions": []
+			},
+			{
+				"property": "f1_totalflow",
+				"friendlyName": "Filter 1 Total Flow",
+				"channel": "f1_totalflow",
+				"type": "Number",
+				"refresh": true,
+				"actions": []
+			},
+			{
+				"property": "f1_totaltime",
+				"friendlyName": "Filter 1 Total Time",
+				"channel": "f1_totaltime",
+				"type": "Number",
+				"refresh": true,
+				"actions": []
+			},
+			{
+				"property": "f1_usedflow",
+				"friendlyName": "Filter 1 Used Flow",
+				"channel": "f1_usedflow",
+				"type": "Number",
+				"refresh": true,
+				"actions": []
+			},
+			{
+				"property": "f1_usedtime",
+				"friendlyName": "Filter 1 Used Time",
+				"channel": "f1_usedtime",
+				"type": "Number",
+				"refresh": true,
+				"actions": []
+			},
+			{
+				"property": "f2_totalflow",
+				"friendlyName": "Filter 2 Total Flow",
+				"channel": "f2_totalflow",
+				"type": "Number",
+				"refresh": true,
+				"actions": []
+			},
+			{
+				"property": "f2_totaltime",
+				"friendlyName": "Filter 2 Total Time",
+				"channel": "f2_totaltime",
+				"type": "Number",
+				"refresh": true,
+				"actions": []
+			},
+			{
+				"property": "f2_usedflow",
+				"friendlyName": "Filter 2 Used Flow",
+				"channel": "f2_usedflow",
+				"type": "Number",
+				"refresh": true,
+				"actions": []
+			},
+			{
+				"property": "f2_usedtime",
+				"friendlyName": "Filter 2 Used Time",
+				"channel": "f2_usedtime",
+				"type": "Number",
+				"refresh": true,
+				"actions": []
+			},
+			{
+				"property": "f3_totalflow",
+				"friendlyName": "Filter 3 Total Flow",
+				"channel": "f3_totalflow",
+				"type": "Number",
+				"refresh": true,
+				"actions": []
+			},
+			{
+				"property": "f3_totaltime",
+				"friendlyName": "Filter 3 Total Time",
+				"channel": "f3_totaltime",
+				"type": "Number",
+				"refresh": true,
+				"actions": []
+			},
+			{
+				"property": "f3_usedflow",
+				"friendlyName": "Filter 3 Used Flow",
+				"channel": "f3_usedflow",
+				"type": "Number",
+				"refresh": true,
+				"actions": []
+			},
+			{
+				"property": "f3_usedtime",
+				"friendlyName": "Filter 3 Used Time",
+				"channel": "f3_usedtime",
+				"type": "Number",
+				"refresh": true,
```

Comment:
Do these have any units?

Reply:
Good question. Not in the spec at least.

I've asked the requestor to look into the app what is the unit.

Reply:
requestor checked, but there is no clear unit in the app visible.